oci/container.go

```diff
@@ -130,6 +132,20 @@ func (c *Container) GetStopSignal() string {
 	return cleanSignal
 }
 
+// GetStopSignalInt returns the container's own stop signal configured from
+// the image configuration or the default one.
+func (c *Container) GetStopSignalInt() syscall.Signal {
```

Comment:
nit: Can we just name this `GetStopSignal`?

Reply:
Oh so you want to overload the existing prototype?

Reply:
Just remembered that Go does not support overloading functions...

Reply:
Done, changed with `StopSignal()`